streamwebs_frontend/streamwebs/models.py

```diff
@@ -41,7 +41,7 @@ class Site(models.Model):
     created = models.DateTimeField(auto_now_add=True)
     modified = models.DateTimeField(auto_now=True)
 
-    objects = SiteManager()
+    test_objects = SiteManager()
```

Comment:
@leian7 is this something we should change for all models or just this one or change GeoManager's `objects` function?